### ISSUE TITLE
[Feature Request] A "show original frame" button in the preview clip window.

### ISSUE THREAD

**Issue by chipweinberger** [RandomEngy/VidCoder#286]
It would also be *amazing* to have the preview window be dynamic as you changed settings.

This would be the most powerful tool for choosing compression settings.

**Comment by RandomEngy** [comment]
I agree, but that's not possible given the limits of the Handbrake engine I use. I think that it's not easy given a lot of the libraries they use don't allow for simple static previews.

**Comment by chipweinberger** [comment]
Truly dynamic preview windows would be a pain to implement, to be sure. But it would be nearly good enough if there was even a long lag, but then let you flip between the (already generated images) of the current settings vs your previous settings.

I was impressed by the preview feature of vidcoder never the less. And the recursive folder support. Video coder is everything handbrake should be.

**Comment by RandomEngy** [comment]
It's more of the process of getting a "true" still frame from the output. You would need to actually encode a small video file then grab a single frame from it. The HandBrake engine doesn't give you that and there is no reliable way to do it outside of there. Plus it would be very slow and would consume a lot of resources as you fiddled with settings.

**Comment by chipweinberger** [comment]
Hmmm, how does the current preview button work?

**Comment by chipweinberger** [comment]
Ah, perhaps I misunderstood what the preview window shows before you click the "encode preview" button. I thought it was still images encoded.

**Comment by RandomEngy** [comment]
Yeah the still images are kind of a "faked up" version of what the output will look like. It reflects cropping, resize, rotation but it's not sent through the actual video encoder.

**Comment by chipweinberger** [comment]
I'd suggest calling it "Crop Region" in the title bar perhaps.
